mackerel/structure_flattens.go

```diff
@@ -334,3 +334,138 @@ func flattenAWSIntegration(awsIntegration *mackerel.AWSIntegration, d *schema.Re
 	}
 	return diags
 }
+
+func flattenDashboard(dashboard *mackerel.Dashboard, d *schema.ResourceData) (diags diag.Diagnostics) {
+	d.Set("title", dashboard.Title)
+	d.Set("memo", dashboard.Memo)
+	d.Set("url_path", dashboard.URLPath)
+	var markdowns []interface{}
```

Comment:
We know that the capacity of these slices does not exceed the length of the `dashboard.Widgets`. How about using `make()` to reserve memory for capacity?